src/Nethermind/Nethermind.Facade/Eth/EthSyncingInfo.cs

```diff
@@ -55,18 +63,33 @@ public SyncingResult GetFullInfo()
             if (_syncConfig.FastSync)
             {
                 if (_syncConfig.DownloadReceiptsInFastSync &&
-                    _receiptStorage.LowestInsertedReceiptBlockNumber > _syncConfig.AncientReceiptsBarrierCalc)
+                    (_receiptStorage.LowestInsertedReceiptBlockNumber == null || _receiptStorage.LowestInsertedReceiptBlockNumber > _syncConfig.AncientReceiptsBarrierCalc))
                 {
-                    return new SyncingResult { IsSyncing = true };
+                    if (_logger.IsInfo) _logger.Info($"Receipts not finished - EthSyncingInfo - HighestBlock: {bestSuggestedNumber}, CurrentBlock: {headNumberOrZero}, AncientReceiptsBarrier: {_syncConfig.AncientReceiptsBarrierCalc}. LowestInsertedBodyNumber: {_blockTree.LowestInsertedBodyNumber} LowestInsertedReceiptBlockNumber: {_receiptStorage.LowestInsertedReceiptBlockNumber}");
```

Comment:
again not needed log as Info?

Reply:
Yes, we were debugging with Kamil. I started with info. That is why it was draft. Changed everything to trace.